lib/aiken/transaction/credential.ak

```diff
@@ -110,3 +110,24 @@ pub type PaymentCredential =
 /// A unique stake pool identifier, as a hash of its owner verification key.
 pub type PoolId =
   Hash<Blake2b_224, VerificationKey>
+
+pub type PubKeyHash =
+  Hash<Blake2b_224, VerificationKey>
+
+pub type ScriptHash =
+  Hash<Blake2b_224, Script>
+
+pub type DatumHashDigest =
+  Hash<Blake2b_256, Data>
+
+pub type DRepCredential {
+  member: Credential,
+}
+
+pub type ColdCommitteeCredential {
+  member: Credential,
+}
+
+pub type HotCommitteeCredential {
+  member: Credential,
+}
```

Comment:
So I went with VerificationKey and VerificationKeyHash since it should align with the already existing functions. The DatumHashDigest I think needs to just get dropped. Datum has a type DatumHash which would have the same name as the actual hash type thus digest was added to that name.